ts/__tests__/Iterable.spec.ts

```diff
@@ -32,6 +32,20 @@ test("set/get email (no token)", () => {
   })
 })
 
+test("setEmail/getEmail_emailAndAuthToken_returnsEmail", () => {
```

Comment:
Is it okay to have `/` in method names?

Reply:
I will update to setEmail_getEmail_emailAndAuthToken_returnsEmail. Thanks 👍